Apps/WebClient/test/functional/e2e/cypress/integration/pages/Registration.js

```diff
@@ -1,21 +1,25 @@
 const { AuthMethod } = require("../../support/constants");
+const registrationPage = "/registration"
 
 describe("Registration Page", () => {
     before(() => {
-        cy.server();
-        cy.fixture("AllDisabledConfig").then((config) => {
-            config.webClient.modules.Comment = true;
-            config.webClient.modules.Medication = true;
-            cy.route("GET", "/v1/api/configuration/", config);
-        });
-        cy.login(
-            Cypress.env("keycloak.unregistered.username"),
-            Cypress.env("keycloak.unregistered.password"),
-            AuthMethod.KeyCloak
-        );
+        cy.readConfig().as("config").then(config => {
+            config.webClient.modules.CovidLabResults = false
+            config.webClient.modules.Comment = true
+            config.webClient.modules.Encounter = false
+            config.webClient.modules.Immunization = false
+            config.webClient.modules.Laboratory = false
+            config.webClient.modules.Medication = true
+            config.webClient.modules.MedicationHistory = false
+            config.webClient.modules.Note = false
+            cy.server();
+            cy.route('GET', '/v1/api/configuration/', config);
+            cy.login(Cypress.env('keycloak.unregistered.username'), Cypress.env('keycloak.unregistered.password'), AuthMethod.KeyCloak, registrationPage);
+        })
     });
 
     it("No sidebar or footer", () => {
+        cy.url().should("include", registrationPage);
```

Comment:
same as above, I'm suggesting using regex, but regex may be overkill for this scenario.

Reply:
I don't think we need regex as it just complicates the test.  We're specifically looking for a url with "registration" and in the context of the test we should not get false positives.